asyncy/omg/OmgExceptions.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+from ..Exceptions import AsyncyError
+
+
+class OmgPropertyTypeNotFoundError(AsyncyError):
+    """
+    Raised when an unknown type is specified which is not
+    recognised as per the OMG specification.
+    """
+    def __init__(self, _type):
```

Comment:
```suggestion
    def __init__(self, type_):
```

Followin PEP8: https://www.python.org/dev/peps/pep-0008/#id36

> single_trailing_underscore_: used by convention to avoid conflicts with Python keyword, e.g.

Reply:
done